### ISSUE TITLE
SwitchResX

### ISSUE THREAD

**Issue by ickc** [caskroom/homebrew-cask#24676]
Hi, for some reason, when installing switchresx:

```bash
brew cask install switchresx
==> Downloading https://www.madrau.com/data/switchresx/SwitchResX4.zip
Already downloaded: /Users/kolen/Library/Caches/Homebrew/Cask/switchresx--4.5.5.zip
==> No checksum defined for Cask switchresx, skipping verification
Error: It seems the Preference Pane source is not there: '/usr/local/Caskroom/switchresx/4.5.5/SwitchResX'
```

If I follow the downloaded file and unzip it, it would install fine. I don't know what cask is expecting here but probably a bug?

**Comment by vitorgalvao** [comment]
You must *always* follow the [reporting bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs) guide. When you opened this issue, the template in place urged you to do so and gave you the relevant links.

The template also stated clearly that

> if it is apparent you ignored the instructions for reporting issues, your issue may be closed without review.

That seems very much to be the case, here, which is why this was closed.

Again, as the template states (emphasis added)

> When the guide isn‘t followed we get the same issues over and over. **Having to repeatedly deal with the same solved and documented problems leads to maintainer burnout and a lot of wasted hours** that could instead have been spent improving Homebrew-Cask itself and fixing real bugs.

I’ll add it is **exhausting**, and when maintainers are exhausted, everyone loses.

A lot of work was put into those guides precisely so we can keep things running smoothly. When users ignore the guides to save a few minutes individually, maintainers lose hours and become fatigued.

I ask that you respect our time and efforts and follow the guides.

**Comment by ickc** [comment]
I did read through it. And then I also try to modify the tap, but it all doesn't work. Only after that I submit what I know.

It is ok to be exhausted, but not ok to assume the others ignored your guide. Sure I might be ignorant that I don't understand what to do. But it doesn't help to be impolite. One problem you guys often have is to assume people who understand English automatically understand things written in English.

**Comment by vitorgalvao** [comment]
> but not ok to assume the others ignored your guide.

When you follow the guide and are not able to make a fix, there’s a link that leads you to a specific template that you can ask for help, and explain why you weren’t able to make the fix. Had you used that, **as the guide tells you to**, I would’ve know, because the template would be there. So no, the guide was not followed.

> But it doesn't help to be impolite.

I was not impolite. That is a canned response, and tailored to ask respectfully for you to follow the guide. If you choose to take that as impolite, there’s not much I can do.

Also, [relevant intrusctions](https://github.com/caskroom/homebrew-cask/blob/master/doc/reporting_bugs/a_cask_fails_to_install.md#source-is-not-there-error), and [fix already done](https://github.com/caskroom/homebrew-cask/pull/24678/files). In the time it took you to reply with that, the fix could’ve been (and was) made.

No point in arguing the point further.

**Comment by ickc** [comment]
If you refer to

> Follow the instructions to submit a fix, and make the change to the cask...

I read that. But I didn't suppose I need to click the link there. I assume it means at the stage I should open an issue. I was puzzled for a moment about "Follow the instructions" but I thought it meant the instructions appears when I open the issue.

So you might consider this as a bug and might be try to improve the wordings there. I read it, but I didn't understand you suppose me to click into it and there's other instructions there. Sometimes even when things written in English doesn't convey what it should be meant.

By the way, canned response is not an excuse, since it is supposed to be written. whether it is composed specifically for me or not is irrelevant.

I knew you fixed it already since I kept trying to make it work and when I uninstalled it and reinstall again it works. I thought it really was an corrupted download but looking carefully it has been modified already. I still don't understand why the previous one doesn't work but I can't replicate the previous state.
